lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -422,12 +422,20 @@ sub load_for_collection {
     return unless $id; # nothing to do
 
     $self->load($collection);
-    my $query = "SELECT " . $self->_columns . "
+    my $query = "SELECT " . $self->_columns . ", ep.value AS prefs_value
```

Comment:
I think I'd prefer renaming `prefs_value` to `show_gravatar` so it's clearer we're just selecting/storing that specific pref there, but just a small nitpick